.github/ISSUE_TEMPLATE/general-feedback.md

```diff
@@ -3,6 +3,8 @@ name: 😡😐😀 General Feedback
 about: For general feedback on the VA API Platform
 ---
 
+## VA Lighthouse API General Feedback
```

Comment:
Could we just make this?: VA Lighthouse General Feedback

Reply:
done